src/features/game/expansion/components/PartyIsland.tsx

```diff
@@ -306,9 +306,11 @@ export const PartyIsland: React.FC = () => {
           }}
         >
           <NPC
-            body="Dark Brown Farmer Potion"
-            onesie="Shark Onesie"
-            hair="Buzz Cut"
+            parts={{
```

Comment:
Not needed, by we now have a `npc.ts` where we store each NPC name and wearable.

This allows us to use them in conversations and announcements.

It would be nice to be able to use some of these NPCs to announce party features ❤️